.pylintrc

```diff
@@ -1,4 +1,4 @@
 [MASTER]
 
 [MESSAGES CONTROL]
-disable = C0114, C0115, C0116, R0903
+disable = C0114, C0115, C0116, R0903
```

Comment:
What has changed here?

If unwanted remove it and then merge.

Reply:
Nothing was changed. I added W0611 and then removed.

Reply:
Then why git diff is showing changes?
Maybe you have added a new line? Can you remove it?

Reply:
No, I didn't add a new line.